test/test-pages/schema-org-context-object/expected-metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "South Korean president’s impeachment fails after ruling party lawmakers walk out",
+  "byline": "Stella Kim, Jennifer Jett",
```

Comment:
It turns out this change doesn't affect any existing test cases, so I added one. This is the key line here. Prior to this change, this would contain data taken from the DOM, starting with "By...".